docs/metadata/git/git-fork.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Fork a repository
+id: git-fork
+description: Source Control
+sidebar_position: 5
+tags:
+  - metadata
+  - Git
+  - fork
+---
+
+This page describes how to fork a repository in cases where you may want to create a copy of the original repository.
+
+## Use fork per user Git storage model
+
+During the creation of Project there is an option to choose fork per usage as Git Storage model.
+When you fork a repository, you create a copy of the original repository (upstream repository) but the repository remains on your GitHub account.
```

Comment:
Prophecy does not create any new repository. We instead ask the user to provide both upstream repo and a fork per user repo, both of which have to be already present. The user needs to make sure that the forked repo they input is actually a fork of whatever they input as upstream, as from Prophecy's perspective these are just 2 repos and we cannot verify if one is actually a fork of the other

Reply:
Done.